app/controllers/patients_controller.rb

```diff
@@ -550,6 +550,15 @@ def patient_diff(patient_before, patient_after)
     diffs
   end
 
+  # Returns case status value(s) of the selected patient(s) so bulk update modal displays correct default value
+  def current_case_status
+    redirect_to(root_url) && return unless current_user.can_edit_patient?
+
+    patient_ids = params[:patient_ids]
+    patients = current_user.viewable_patients.where(id: patient_ids)
```

Comment:
@holmesie I think we need to get in the habit of using `current_user.patients` in case this is ever available to enrollers.

Reply:
👍